index.bs

```diff
@@ -544,16 +544,18 @@ tracks connected to it, possibly including the {{MediaStreamTrackGenerator}}
 itself, are ended and the `ended` event is fired on them.
 
 # Examples # {#examples}
+## Video Processing
 Consider a face recognition function `detectFace(videoFrame)` that returns a face position
 (in some format), and a manipulation function `blurBackground(videoFrame, facePosition)` that
-returns a new VideoFrame similar to the given `videoFrame`, but with the non-face parts blurred.
+returns a new VideoFrame similar to the given `videoFrame`, but with the non-face parts blurred. Then display the before and after tracks on video elements.
 
 <pre class="example">
-let stream = await getUserMedia({video:true});
-let videoTrack = stream.getVideoTracks()[0];
-let trackProcessor = new MediaStreamTrackProcessor(videoTrack);
-let trackGenerator = new MediaStreamTrackGenerator();
-let transformer = new TransformStream({
+async function main() {
```

Comment:
It looks like this line was unintentional. If not, indentation is wrong and you're missing a close brace.